components/GameCard.js

```diff
@@ -2,12 +2,18 @@ import styled from "styled-components";
 import Link from "next/link";
 import FavouriteButton from "./FavouriteButton";
 
-export default function GameCard({ game }) {
+export default function GameCard({ game, isFavorite, toggleFavorite }) {
+  const checkIsFavorite = isFavorite(game._id);
+
   return (
     <StyledCard>
       <StyledFavoriteButton>
-        <FavouriteButton />
+        <FavouriteButton
+          isFavorite={checkIsFavorite}
+          toggleFavorite={() => toggleFavorite(game._id)}
+        />
       </StyledFavoriteButton>
```

Comment:
please merge the styling of both elements or call one of the components "FavoriteButtonWrapper"